src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -18,6 +18,23 @@ namespace Microsoft.DotNet.GenAPI
     /// </summary>
     public static class GenAPIApp
     {
+        // Attributes that can work with local definition and no runtime support to light up
+        // public API behavior in the language / compiler that are not defined in all supported frameworks
+        // see https://github.com/dotnet/roslyn/blob/859f94ef2d8bf88527217bc9ad7661b6fbdf33a9/src/Compilers/Core/Portable/Symbols/Attributes/AttributeDescription.cs#L343
+        private static readonly string[] s_compilerAttributes =
```

Comment:
@333fred @jcouv @jaredpar - would be interested in your thoughts of what API we consider "OOB"able in the set of well known types.  I filed https://github.com/dotnet/roslyn/issues/71790 to track a roslyn API to help with this task.

Reply:
Could you please clarify what OOB means?

Reply:
API that a user project might decide to define in their own assembly in order to use the language feature.

Reply:
I guess I'm not sure why you'd want to treat these differently than any other internal API?

Reply:
Looking more closely - many of these will automatically be synthesized by the compiler when used - so we don't need to special case them:
https://github.com/dotnet/roslyn/blob/291d5d70fb062d1c8caf324b9369c535befaf6ed/src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs#L1492

So this is really about features where they are directly used by the library author and not synthesized by the compiler.

Reply:
Talking with Eric offline, I'd like to propose a namespace filter instead: if we just include internal things from `System.Runtime.CompilerServices`, `System.Runtime.InteropServices`, and `System.Diagnostics.CodeAnalysis`, I believe that would be a more robust approach that should handle all of these cases automatically.

Reply:
Fred and I chatted offline and we think we'll instead try a namespace inclusion filter.  We'll experiment with including *everything* from under `System.Runtime.CompilerServices`, `System.Runtime.InteropServices`, and `System.Diagnostics.CodeAnalysis` modulo what gets excluded from other means.  I can try this with SBRP packages and see what the effect is.  If it's net positive we can go that route - if we need a lot of exclusions we might just leave out the defaults and ask users to configure the inclusions where needed.